docs/administration-guide/gaffer-deployment/gaffer-docker/how-to-run.md

```diff
@@ -0,0 +1,270 @@
+# How to Deploy with Accumulo on Docker
+
+After reading the [previous page](./gaffer-images.md) you should have a good
+understanding of what images are available for Gaffer and how to configure them
+to you needs. However, before running a deployment backed by Accumulo you will
+need to know a bit of background on Hadoop to understand how the data will scale
+and be distributed.
+
+Usually when deploying a container image you simply run the image and everything
+is contained locally to the container (hence the name). For larger scale graphs
+this less desireable as we will usually want to be able to scale and load
+balance the storage based on the volume of data; this is where Hadoop comes in.
+
+## What is Hadoop/Accumulo?
```

Comment:
This is a nice explanation of Hadoop, but I'm not sure it should live on this page.

Reply:
The idea was you really should know about it before doing a deployment I didn't think it was big enough to warrant its own page

Reply:
Maybe it could go on the Accumulo Store page?